sdk/storage/azure_storage_blob/tests/framework/test_account.rs

```diff
@@ -0,0 +1,53 @@
+use std::sync::{Arc, Once};
+
+use azure_core::{credentials::Secret, TransportOptions, Uuid};
+use azure_core_test::TestContext;
+use azure_data_cosmos::{CosmosClientOptions, Query}; // Get rid of this
+use futures::StreamExt;
+use reqwest::ClientBuilder;
+use time::{macros::format_description, OffsetDateTime};
+
+/// Represents a Azure Storage account for testing purposes.
+///
+/// A [`TestAccount`] serves two main purposes:
+/// * Managing connection information to make it easy to connect to the account the tests are using.
+/// * Provide a unique ID to each test and automatic clean-up of resources.
+pub struct TestAccount {
+    pub context_id: String,
+    endpoint: String,
+    key: Secret,
+    options: TestAccountOptions,
+}
+
+#[derive(Default)]
+pub struct TestAccountOptions {
+    pub allow_invalid_certificates: Option<bool>,
+}
+static TRACING: Once = Once::new();
+
+// Assuming we will want to change this over to DefaultAzureCredential
+impl TestAccount {
```

Comment:
I am assuming the way we will use this is sort of have it set everything up, and so all values used for testing should be sourced from the `TestAccount` so that it is standardized and will set us up for success in terms of sanitization etc. and matching CI account configuration.

Reply:
N/A